gateway/src/main/java/com/hurence/webapiservice/historian/impl/SolrHistorianServiceImpl.java

```diff
@@ -341,6 +347,50 @@ public HistorianService getAnnotations(JsonObject params, Handler<AsyncResult<Js
         return this;
     }
 
+    @Override
+    public HistorianService addTimeSeries(JsonArray timeseries, Handler<AsyncResult<JsonObject>> resultHandler) {
+
+        Handler<Promise<JsonObject>> getMetricsNameHandler = p -> {
+            try {
+                JsonObject response = new JsonObject();
+                Collection<SolrInputDocument> documents = new ArrayList<>();
+                int numChunk = 0;
+                int numPoints = 0;
+                for (Object TimeseriesObject : timeseries) {
```

Comment:
TimeseriesObject is not a standard variable name !

Reply:
change it to timeserieObject